backend/dataall/modules/dataset_sharing/services/share_managers/s3_access_point_share_manager.py

```diff
@@ -12,15 +12,17 @@
 from dataall.modules.dataset_sharing.db.share_object_models import ShareObject
 from dataall.modules.dataset_sharing.services.dataset_alarm_service import DatasetAlarmService
 from dataall.modules.dataset_sharing.db.share_object_repositories import ShareObjectRepository
+from dataall.modules.dataset_sharing.services.share_managers.share_manager_utils import ShareManagerUtils
 
 from dataall.modules.datasets_base.db.dataset_models import DatasetStorageLocation, Dataset
 
 logger = logging.getLogger(__name__)
 ACCESS_POINT_CREATION_TIME = 30
 ACCESS_POINT_CREATION_RETRIES = 5
+IAM_ACCESS_POINT_ROLE_POLICY = "targetDatasetAccessControlPolicy"
```

Comment:
Should we also add another variable `DATAALL_ALLOW_OWNER_SID="AllowAllToAdmin"` similar to have we do with `IAM_ACCESS_POINT_ROLE_POLICY` here and with `DATAALL_ALLOW_OWNER_SID` in `s3_bucket_share_manager.py`

Reply:
Makes sense! Done.